### PR TITLE
[EGD-7927] Long track attributes/names drawing

### PR DESCRIPTION
**Pull request by tomaszkrosnowski**
When playing tracks with long names they were incorrectly drawn in MP.
Now the beginning of string + elipsis is shown.

### CODE REVIEW

**Review by PrzeBrudny — COMMENTED** [review]
If these changes are not critical please wait for Label refactor -> it should start next week.

File: module-apps/application-music-player/windows/MusicPlayerMainWindow.cpp [hunk 1]
```diff
@@ -337,13 +337,13 @@ namespace gui
...
         textBox->setAlignment(Alignment(Alignment::Horizontal::Left, Alignment::Vertical::Center));
-        descriptionText = new Text(textBox, 0, 0, 0, 0);
+        descriptionText = new Label(textBox, 0, 0, 0, 0);
         descriptionText->setMinimumSize(trackInfo::width - trackInfo::height - trackInfo::internalMargin,
                                         trackInfo::height);
         descriptionText->setAlignment(Alignment(Alignment::Horizontal::Left, Alignment::Vertical::Center));
-        descriptionText->setTextType(TextType::SingleLine);
-        descriptionText->setEditMode(EditMode::Browse);
         descriptionText->setFont(style::window::font::small);
+        descriptionText->setPenFocusWidth(0);
```
Comment (PrzeBrudny) [inline module-apps/application-music-player/windows/MusicPlayerMainWindow.cpp:345]: setEdges for None is enough.

Comment (tomaszkrosnowski) [reply module-apps/application-music-player/windows/MusicPlayerMainWindow.cpp]: done

**Review by kkleczkowski — APPROVED** [review]

**Review by Maciej-Mudita — APPROVED** [review]

**Comment by tomaszkrosnowski** [comment]
> If these changes are not critical please wait for Label refactor -> it should start next week https://appnroll.atlassian.net/browse/EGD-3544

Label is only widget that supports elipsis for now, and from what we talked it will remain in code..

**Review by PrzeBrudny — APPROVED** [review]

**Comment by tomaszkrosnowski** [comment]
Smoke tests completed https://appnroll.atlassian.net/browse/EGDT-1982

**Pull request merged**